app/marsRoverPath/styles.css

```diff
@@ -7,10 +7,19 @@ body {
 div {
   width: 100%;
   height: 100%;
+  position: relative;
 }
 
 img {
   width: 100%;
   height: 100%;
   object-fit: contain;
 }
+
+#rover-sprite {
+  width: 6%;
+  height: auto;
+  position: absolute;
+  top: calc((10 / 946) * 100%);
+  left: calc((1150 / 1920) * 100%);
```

Comment:
Does this bit always work, including if you change the screen size to be wider or less wide?

Reply:
This has been refactored so the position is done programmatically.